cmd/ceph/ceph.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I kept main.go under cmd folder and all other commands inside `commands` folder otherwise everything go file will be under main package and now current structure seems okay to me